storage/test/storage/HashMapStorageTest.java

```diff
@@ -0,0 +1,32 @@
+package storage;
+
+import model.Resume;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import storage.map.MapStorage;
+
+public class HashMapStorageTest extends ArrayStorageTest {
+
+    public HashMapStorageTest() {
+        super(new MapStorage());
+    }
+
+    @Override
+    void save_throwsAddToFullStorageException() {
```

Comment:
LSP violation here. Map is not an array with limits.

Reply:
Done